backend/api/views/quiz.py

```diff
@@ -1,10 +1,110 @@
+import json
+
+from django.conf import settings
 from rest_framework.decorators import api_view
+from rest_framework import status
+from pymongo.collection import ReturnDocument
 
+from ..validators import SchemaValidator
 from ..client import client
-from ..utils import JsonResponse
+from ..utils import JsonResponse, get_sort_order_from_request, get_new_id
+from ..exceptions import InvalidData
+
+
+__quizzes_collection = client.collection("quizzes")
```

Comment:
Why the prefix `__` ?

Reply:
I try to use it as markers for objects that souldn't be used outside of a module.
Do you prefer I removed them ?

Reply:
Hum.. 

- First, I'm not really convinced by the fact that you're defining the collection a t the top level. If it were a SQL database, I think you could get errors since it is not scoped.
- Second, having a class would resolve the first point. And you can use mangling in your class if you want (but I prefer `quizzes.find_one()` than `__quizzes_collection.find_one()`)